### PR TITLE
[YD-408] Creating the sample app

### PR DESCRIPTION
**Pull request by eduardoyoti**
In this pull request we are creating a sample app that uses the Yoti Docs SDK.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    ext.kotlin_version = '1.3.31'
```
Comment (cquemin) [inline build.gradle]: `v1.3.50` ?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,161 @@
...
+| 6000              | SDK is out-of-date - please update the SDK to the latest version          | No |
+| 6001              | Unexpected internal error          | No |
+| 6002              | Unexpected document scanning error          | No |
+
+
+## Customisation
+You can customise the appearance of the screens of the SDK by overriding some styles.
+
+### Font Family and Font Colour
+In order to change the font family and the font colour you just need to define the following styles:
```
Comment (mariasenosiain) [inline README.md]: This fonts are part of the Yoti Android UI styles which are used by the SDK, the SDK ones are: 
YotiDocs.TextView.Title
YotiDocs.TextView.Header
YotiDocs.TextView.Text
YotiDocs.TextView.Link

File: README.md [hunk 3]
```diff
@@ -1,2 +1,161 @@
...
+If you just want to change the colour of the font, then override the following colour:
+
+```xml
+<color name="yotiDocs_colorFont">
+```
+
+### Primary Colours
+In order to change the colours of the different elements of the screens you just need to declare the following colours:
+
+- For the main app branding colour:
```
Comment (mariasenosiain) [inline README.md]: Here we can also add a mention to personalise the font color with `yotiDocs_colorFont`

File: app/src/main/res/values/styles.xml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+<resources>
+
+    <!-- Base application theme. -->
+    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+        <!-- Customize your theme here. -->
+    </style>
+
+    <!-- Override Yoti fonts style to use GothicA1 -->
+    <style name="Yoti.v3.Font.DisplayBold">
```
Comment (mariasenosiain) [inline app/src/main/res/values/styles.xml]: We also have to correct the font overriding here. The fonts that can be overridded are: 
- YotiDocs.TextView.Title
- YotiDocs.TextView.Header
- YotiDocs.TextView.Text
- YotiDocs.TextView.Link

File: README.md (hunk 3 above)
Comment (eduardoyoti) [reply README.md]: That's been mention in the font section above. 👍

**Pull request merged**